Sources/SwiftDocC/Utility/MarkupExtensions/ListItemExtractor.swift

```diff
@@ -106,6 +114,13 @@ extension Sequence where Element == InlineMarkup {
         return nil
     }
     
+    func extractPossibleValue() -> PossibleValue? {
+        if let (value, _, content, _) = splitNameAndContent() {
```

Comment:
New code should capture and use the `nameRange` and `itemRange`. Otherwise diagnostics for this content can't be associated with the file where the content comes from.

It would also be good to add a test that verifies that diagnostics _in the possible value content_ are presented with the relevant source and range information.